quill-core/src/main/scala/io/getquill/quotation/Parsing.scala

```diff
@@ -353,11 +353,15 @@ trait Parsing extends SchemaConfigParsing {
   private def is[T](tree: Tree)(implicit t: TypeTag[T]) =
     tree.tpe <:< t.tpe
 
+  private def isTraversable(tree: Tree) = {
+    tree.tpe <:< typeOf[Traversable[_]]
```

Comment:
@fwbrasil If we define an object like this
```scala
object Foo {
  def apply(a: Int*): Seq[Int] = {
    a.map(_ * 2)
  }
}
```
It will be parsed too.